.buildkite/x-pack/pipeline.xpack.agentbeat.yml

```diff
@@ -88,6 +94,58 @@ steps:
         notify:
           - github_commit_status:
               context: "agentbeat: Packaging"
+      - label: ":linux: Agentbeat packaging linux/amd64 FIPS"
+        key: "agentbeat-package-linux-x86-fips"
+        depends_on:
+          - mandatory-win-2022-system-tests
+        env:
+          PLATFORMS: "${PLATFORMS_AMD64_FIPS}"
+          SNAPSHOT: true
+          FIPS: "true"
```

Comment:
```suggestion
          FIPS: true
```

Reply:
Done in f35490f4f2b69447ed85f6716817e581bee82e95